assets/script.js

```diff
@@ -82,4 +82,20 @@ if (document.location.pathname.indexOf('fullcpgrid') > -1) {
 // Current year of footer
 const currentYearSpan = document.getElementById('currentYear')
 const currentYear = new Date().getFullYear()
-currentYearSpan.innerText = currentYear
\ No newline at end of file
+currentYearSpan.innerText = currentYear
+
+// Search bar 
+function search_card() {
+  let input = document.getElementById('searchbar').value
+  input=input.toLowerCase();
+  let x = document.getElementsByClassName('card');
+    
+  for (i = 0; i < x.length; i++) { 
+      if (!x[i].innerHTML.toLowerCase().includes(input)) {
+          x[i].style.display="none";
+      }
+      else {
+          x[i].style.display="card";                 
```

Comment:
This is a typo you probably noticed, the display here should be `flex`and not `card`